app/views/browse/note.html.erb

```diff
@@ -17,6 +17,11 @@
       <br/>
       <%= note_event(@note.status, @note.closed_at, @note_comments.last.author) %>
     <% end %>
+    <% if current_user && current_user != @note.author %>
+      <%= link_to new_report_url(reportable_id: @note.id, reportable_type: @note.class.name), :title => t('browse.note.report') do %>
+          &nbsp;&#9872;
```

Comment:
I really don't like the use of that unicode flag character - is anybody likely to recognise what it means? I don't think it's commonly used elsewhere? There's also the question of how widely it will render correctly...

If we can't come up with anything better then maybe we should just use text like `Report user` etc instead?